sections/Chapter 1 - Introduction to knox/40_Agile-Cooperation.tex

```diff
@@ -18,4 +18,13 @@ \subsection{Roles and Events}\label{roles_and_events}
 All groups made a backlog for the project together, however, all tasks were somewhat loosely defined, and it was up to the assigned groups to specify them. Sprint planning was done during the Scrum-of-Scrum meetings, which were held weekly throughout the project. Sprint reviews began during the third sprint and were made with a PowerPoint containing the finished and in-progress tasks of the groups. A component diagram of the Knox project was also updated each week. Sprint retrospective was held by the Scrum masters of all groups between sprints.
 
 \subsection{Our Agile Approach}\label{our_agile_approach}
-\todo{write about our own agile approach - the initial methods used and such.}
+\textit{This section is not part of the collective writing.}
+
+In our group, we have strived to utilize as much of Scrum as we could.
+As is the case in Scrum, a given group has one product owner, one Scrum master, and some developers.
+
+The product owner is, as the name implies, responsible for the product. In practice, this makes them accountable for maximizing the value of the product. To this end, the product owner manages the product backlog, and is responsible for the product vision. As we participate in a Scrum-of-Scrum hierarchy, the product owner is responsible for reporting progress on the project to the Knox product owner as well.
+
+The Scrum master is responsible for enforcing Scrum. More importantly, the Scrum master is responsible for guiding the team towards continuous improvement, which leads to increased velocity and happiness.
+
+Our general approach imitates the Scrum method. First, we have a product backlog filled with potential improvements to the project. Each sprint, we, the scrum team, take the most important of these and plan an increment based on them. We work towards this increment during the sprint. Each day, we have a short Daily Scrum, where we discuss progress and eventual obstacles. At the end of the sprint, we have a Sprint Review, where we discuss the progress of the sprint and should happen next. Finally, we have a Sprint Retrospective, where we discuss how the sprint went and what we learned.
```

Comment:
A short text describing that each sprint section will follow this structure but will also contain information about agile methodology, collaboration with the other knox groups, and database/server management.